app/gateway/database/postgres/accounts/create_account_test.go

```diff
@@ -59,22 +58,30 @@ func Test_Create(t *testing.T) {
 				}
 			},
 			want: account.Account{
-				ID:      "",
-				Name:    "",
-				CPF:     "",
-				Balance: 0,
+				ExternalID: "",
+				Name:       "",
+				CPF:        "",
+				Balance:    0,
 			},
 			wantErr: true,
 		},
 	}
 
 	for _, test := range testCases {
 		t.Run(test.name, func(t *testing.T) {
+
 			TruncateTable(database)
 			if test.runBefore != nil {
 				test.runBefore(database)
 			}
 			got, err := accountRepository.Create(ctx, test.input)
+
+			if err != nil {
+				test.want.CreatedAt = got.CreatedAt
+				test.want.ID = got.ID
+				test.want.ExternalID = got.ExternalID
+			}
```

Comment:
Acredito que essa asserção deveria acontecer se o erro esperado for nil. Acho isso pq, pelo o que entendi, esses dados fornecidos pelo banco devem ser fornecido quando o esperado for nenhum erro. Tipo, se acontecer um erro quando não deveria essa asserção vai ficar faltando.

Reply:
Exato, foi um erro na hora que estava reproduzindo o código hehe Já tá arrumado nos commits novos.